api/src/pcapi/core/educational/api/adage.py

```diff
@@ -7,6 +7,7 @@
 from pcapi.core.educational import models as educational_models
 from pcapi.core.educational import repository as educational_repository
 from pcapi.core.educational.api.venue import get_relative_venues_by_siret
+from pcapi.core.mails.transactional.educational.eac_sending_offerer_activation import send_eac_offerer_activation_email
```

Comment:
(détail) On a des raccourcis d'import dans `pcapi/core/mails/__init__.py` qui permettent de s'éviter ce genre d'import à rallonge. Si tu y rajoutais un raccourci pour ta nouvelle fonction, ça donnerait : 

```suggestion
from pcapi.core.mails.transactional import send_eac_offerer_activation_email
```